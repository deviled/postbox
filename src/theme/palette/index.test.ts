import { palette } from '.';

describe('palette', () => {
  it('should match snapshot', () => {
    expect(palette).toMatchSnapshot();
  });
});
