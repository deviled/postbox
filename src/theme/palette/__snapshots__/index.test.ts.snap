// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`palette should match snapshot 1`] = `
Object {
  "background": "lightgray",
  "error": "darkred",
  "info": "aqua",
  "text": "black",
}
`;
